app/views/responsiblepeople/detailed_answers.scala.html

```diff
@@ -70,6 +71,15 @@
     @heading("title.cya", "summary.responsiblepeople")
 }
 
+@headingFunction = { @businessTypes.map { types =>
```

Comment:
maybe it would be better to change the name of above from headingFunction to something more descriptive to know what it's doing (more verb-like identifier) like: getQuestion, getHeading, getDetailedAnswerTitle... or any other you like :)

Reply:
Done